eclipse-orion/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM kdelfour/supervisor-docker
+MAINTAINER Brian Dant <brian@appsembler.com>
+
+RUN apt-get update 
```

Comment:
In a Dockerfile, it's always a good idea to combine the `apt-get update and apt-get install` steps into a single `RUN` line. Otherwise, you end up in a situation where docker has cached the build up to `apt-get update` but wants to re-run the `install` step and it fails because the apt cache is out of date.

I would combine these and move the `apt-get clean` from below into it as well (helps keep image size down):

Something like:

    RUN apt-get update -y && \
             apt-get install -y \
                          build-essential \
                          curl \
                          g++ \
                          git \
                          libssl-dev apache2-utils \
                          libxml2-dev \
                          maven \
                          sshfs && \
           apt-get clean && rm -rf /var/lib/apt/lists/*

(putting all the packages on individual lines makes for easier diffs later on)